code/_onclick/hud/rendering/plane_masters/_plane_master.dm

```diff
@@ -64,9 +67,12 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/plane_master)
 	/// If this plane master is outside of our visual bounds right now
 	var/is_outside_bounds = FALSE
 
-/atom/movable/screen/plane_master/Initialize(mapload, datum/hud/hud_owner, datum/plane_master_group/home, offset = 0)
+/atom/movable/screen/plane_master/Initialize(mapload, datum/hud/hud_owner, datum/plane_master_group/home, offset = 0, relay_loc)
 	. = ..()
 	src.offset = offset
+	if(relay_loc)
+		src.relay_loc = relay_loc
```

Comment:
the plane master should not have an opinion about the relay loc
it knows what group owns it, so we can just use the relay_loc from its parent home rather then trying to pipe it down like this

Reply:
removed